Catappult/appcoins-wallet-android
# [feature/APPC-3221] VIP splash screen

**What does this PR do?**

   Implements the splash screen animation for VIP users and navigates after the animation completes

**Database changed?**

   No

**Where should the reviewer start?**

- SplashExtenderFragment.kt

**How should this be manually tested?**

  - Change to a vip wallet
  - Close the app
  - Open the app again and check the animation and the navigation after

**What are the relevant tickets?**

  https://aptoide.atlassian.net/browse/APPC-3221

**Code Review Checklist**
